### PR TITLE
Add XRS to the archived data, optimize downloads and processing

### PR DESCRIPTION
**Pull request by oharao**
Add XRS to the archived data, optimize downloads of goes and processing. Additionally removed trailing data points causing a drop off in plots.

### CODE REVIEW

File: supersid/config/config.py [hunk 1]
```diff
@@ -10,11 +10,11 @@
...
 ### [PARAMETERS] ###
 
-# Path to the file directory locating the raw csv data you wish to process.
-data_path = 'C:/Users/oscar/OneDrive/Desktop/temp'
+# Paths to the file directories locating the raw csv data you wish to process.
+data_path = ['C:/Users/oscar/Desktop/temp']
 
 # Output path of archive - archive will be generated within the specified directory.
-archive_path = 'C:/Users/oscar/OneDrive/Desktop/SuperSid/data'
+archive_path = 'C:/Users/oscar/Desktop/SuperSid/data'
```
Comment (hayesla) [inline supersid/config/config.py:17]: maybe I'm missing something here - is this on both machines? and why are you moving it out of OneDrive?

File: supersid/vlfclient.py [hunk 2]
```diff
@@ -104,190 +110,183 @@ def norm_data(self, df):
...
         Parameters
         ----------
-        parameters : dict
+        header : dict
             Dictionary containing observation parameters, eg. transmitter freq.
         data : object
             Pandas dataframe containing csv data.
         file_path : str
             Path to csv file.
+        gl : pandas.timeseries
```
Comment (hayesla) [inline supersid/vlfclient.py]:
```suggestion
        gl : pandas.Series
```

File: supersid/vlfclient.py [hunk 3]
```diff
@@ -104,190 +110,183 @@ def norm_data(self, df):
...
+            GOES XRS Long data.
+        gs : pandas.timeseries
```
Comment (hayesla) [inline supersid/vlfclient.py]:
```suggestion
        gs : pandas.Series
```

**Review by hayesla — APPROVED** [review]
This looks great Oscar - some minor comments and then its good to merge

File: supersid/config/config.py (hunk 1 above)
Comment (oharao) [reply supersid/config/config.py:17]: So this is my local machine config (weirdly my desktop as two locations) and wished to make it more accessible, each machine has its own local branch which changes those though. Couldn't think of a better way to do it any suggestions?

Comment (hayesla) [reply supersid/config/config.py:17]: ah ok - so when you install this on the different machines, it changes this path?

Comment (oharao) [reply supersid/config/config.py:17]: Yes, there is like a commit that always sits on top just changing these paths based on the os machine file structure.

Comment (hayesla) [reply supersid/config/config.py:17]: ah ok perfect. This is good for now - was just curious :)

**Pull request merged**